server/scopeserver/dataserver/utils/search_space.py

```diff
@@ -154,14 +154,9 @@ def add_markers(self, element_type="regulon_target") -> None:
                     else:
                         self.search_space_dict[key] = [regulon]
         if element_type == "marker_gene":
-            searchable_clustering_ids = [
-                x.split("_")[-1] for x in loom.ra.keys() if bool(re.search("ClusterMarkers_[0-9]+$", x))
-            ]
-            for clustering in searchable_clustering_ids:
-                clustering = int(clustering)
-                for cluster in range(len(self.meta_data["clusterings"][clustering]["clusters"])):
-                    genes = self.loom.get_cluster_marker_genes(clustering, cluster)
-                    for gene in genes:
+            for clustering in self.meta_data["clusterings"]:
+                for cluster in clustering["clusters"]:
+                    for gene in self.loom.get_cluster_marker_genes(clustering["id"], cluster["id"]):
```

Comment:
Any changes in the search space should also increment `CURRENT_SS_VERISON` on line 12

Reply:
Why would this change warrant a `CURRENT_SS_VERSION` bump?